include/core/tracepoint.h

```diff
@@ -31,7 +31,9 @@
 #define TRACEPOINT_H
 
 #include "lf_types.h"
+#if !defined(PLATFORM_FLEXPRET)
```

Comment:
Not great to have this here. See my comment above. Maybe this include statement should be in platform files instead.